{"name": "test03_mis_audit_article", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <scripts.test02_mis.TestMis object at 0x0000025BF090E730>\n\n    def test03_mis_audit_article(self):\n        r = self.mis.mis_article_audit()\n        try:\n>           Tools.common_assert(r)\n\nscripts\\test02_mis.py:34: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'tools.tool.Tools'>, response = <Response [403]>, statues_code = 201\n\n    @classmethod\n    def common_assert(cls, response, statues_code=201):\n        # 断言状态码\n        #assert statues_code == response.statues_code\n        # # 断言响应信息\n        log.info(\"正在调用公共断言方法。。。。。。\")\n>       assert \"OK\" == response.json().get(\"message\")\nE       AssertionError\n\ntools\\tool.py:20: AssertionError"}, "attachments": [{"name": "log", "source": "e8d5255e-c567-4120-a161-a3d68b17e73e-attachment.txt", "type": "text/plain"}], "start": 1617425150516, "stop": 1617425150611, "uuid": "5bac8807-4c20-4ecc-a422-6adacd5ec464", "historyId": "06c46721f8b4c1013c7f0091b51c4ceb", "testCaseId": "ca7660fd3a8bd8fa63dc8a67ce252269", "fullName": "scripts.test02_mis.TestMis#test03_mis_audit_article", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test02_mis"}, {"name": "subSuite", "value": "TestMis"}, {"name": "host", "value": "fanxx"}, {"name": "thread", "value": "17644-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test02_mis"}]}